File: 배열/leetcode1470(Easy)-Shuffle the Array.py
# 61ms(79.49%), 14.2MB(33.52%)
class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        result = []
        for i in range(n):
            result.append(nums[i])
            result.append(nums[i + n])
        
        return result

File: 배열/leetcode1470(Easy)-Shuffle the Array.js
// 76ms(62.90%), 44.1MB(81.48%)
/**
 * @param {number[]} nums
 * @param {number} n
 * @return {number[]}
 */
var shuffle = function (nums, n) {
  const result = []
  for (let i = 0; i < n; i += 1) {
    result.push(nums[i])
    result.push(nums[i + n])
  }
  return result
}
